src/java/picard/illumina/IlluminaBasecallsToFastq.java

```diff
@@ -88,9 +118,9 @@
     @Option(doc = "Lane number. ", shortName = StandardOptionDefinitions.LANE_SHORT_NAME)
     public Integer LANE;
 
-    @Option(doc = "The prefix for output fastqs.  Extensions as described above are appended.  Use this option for " +
-            "a non-barcoded run, or for a barcoded run in which it is not desired to demultiplex reads into separate " +
-            "files by barcode.",
+    @Option(doc = "The prefix for output FASTQs.  Note, you should actually use a file path here to direct the output to a location outside of the working directory.  " +
+            "Extensions as described above are appended.  Use this option for a non-barcoded run, or for a barcoded run in which it is not " +
```

Comment:
"Extensions as described above are appended." -> I don't actually understand this sentence. I know it's not your text originally, but boy scout rule -> if you touch it, make it better.